tests/acceptance/features/apiSpaces/resharing.feature

```diff
@@ -0,0 +1,85 @@
+@api @skipOnOcV10
+Feature: Resharing
+  It is possible to reshare files
+
+  Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+  See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+      | Carol    |
+      | Damian   |
+    And user "Alice" has created folder "folder"
+    And user "Alice" has shared folder "folder" with user "Brian" with permissions "31"
+    And user "Brian" accepts share "/folder" offered by user "Alice" using the sharing API
+    And as "Brian" folder "Shares/folder" should exist
```

Comment:
not sure that we need check it in GivenSteps. If this folder is not present, the test will break in the next step. in scenario we can use this checking